src/formatting/missed_spans.rs

```diff
@@ -63,13 +64,15 @@ impl<'a> FmtVisitor<'a> {
             }
             let indent = this.block_indent.to_string(config);
             this.push_str(&indent);
-        })
+        });
+        self.normalize_vertical_spaces = false;
```

Comment:
Sorry for the borderline pun, but could we normalize the variable/function names and replace the old `vertical_spaces` with `newline` consistently everywhere?

```suggestion
        self.normalize_newline_counts = false;
```